back/boxtribute_server/business_logic/metrics/queries.py

```diff
@@ -16,3 +20,70 @@ def resolve_metrics(*_, organisation_id=None):
 
     # Pass organisation ID to child resolvers
     return {"organisation_id": organisation_id}
+
+
+@public_query.field("newlyRegisteredBeneficiaryNumbers")
+def resolve_newlyRegisteredBeneficiaryNumbers(*_):
+    now = datetime.now()
+
+    # Last month
+    first_day_this_month = now.replace(day=1)
+    last_day_of_last_month = first_day_this_month - timedelta(days=1)
+    first_day_of_last_month = last_day_of_last_month.replace(day=1)
+
+    # Last quarter
+    curr_quarter = (now.month - 1) // 3 + 1
+    last_quarter = curr_quarter - 1
+
+    year = now.year
+    if last_quarter == 0:
+        last_quarter = 4
+        year -= 1
+
+    first_month_last_quarter = 3 * (last_quarter - 1) + 1
+    quarter_start = date(year, first_month_last_quarter, 1)
+
+    if last_quarter == 4:
+        next_quarter_start = date(year + 1, 1, 1)
+    else:
+        next_quarter_start = date(year, first_month_last_quarter + 3, 1)
+
+    quarter_end = next_quarter_start - timedelta(days=1)
+
+    # Last year
+    start_last_year = date(now.year - 1, 1, 1)
+    end_last_year = date(now.year - 1, 12, 31)
+
+    # Query counts
+    last_month_count = (
+        Beneficiary.select()
+        .where(
+            (Beneficiary.created_on >= first_day_of_last_month)
+            & (Beneficiary.created_on <= last_day_of_last_month)
+        )
+        .count()
```

Comment:
this block repeats two times, please add a small function `number_of_beneficiaries_registered_between(start, end)`.
Make the function return 0 if the result of count() would be None (to simplify lines 86-88)